### PR TITLE
Adding ability to provide own picture for facebook share dialogs

### PR DESCRIPTION
**Pull request by rhwilburn**
Modifying @types/facebook-js-sdk

Adding support for custom thumbnail images for using the facebook share
dialog. Facebook will provide default images for your dialog if you
don't specify your own. This allows you to override this. It works but
it isn't documented under facebook documentation unfortunately.

Adding fix for #15908 in the master repo.

========================================================

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ Can't] Provide a URL to documentation or source code which provides context for the suggested changes: Can't because I can't find documentation for this. It does work however. I have attached screenshot showing this. You can see the stick man is clearly not what facebook would generate for that web page by default
image

- [ N/A?] Increase the version number in the header if appropriate.
- [ N/A? ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

### CODE REVIEW

File: types/facebook-js-sdk/index.d.ts [hunk 1]
```diff
@@ -113,6 +113,7 @@ declare namespace facebook {
     interface ShareDialogParams extends DialogParams {
         method: 'share';
         href: string;
+        picture: string;
         hashtag?: string;
```
Comment (amritk) [inline types/facebook-js-sdk/index.d.ts]: picture should be optional here

**Review by amritk — CHANGES_REQUESTED** [review]

**Comment by aozgaa** [comment]
Can you make the requested changes, @rhwilburn?

If there's no activity on this PR within a week or two, we will close it.

**Comment by rhwilburn** [comment]
I have made the suggested change I believe. Apologies on the delay.

**Comment by amritk** [comment]
👍

**Pull request merged**